logging.yaml

```diff
@@ -0,0 +1,27 @@
+version: 1
+disable_existing_loggers: true
+loggers:
+  app_logger:
+    level: INFO
+    handlers: [console]
+    propagate: no
+handlers:
+  console:
+    class: logging.StreamHandler
+    level: INFO
+    formatter: plaintext
+    stream: ext://sys.stdout
+  file:
+    class: logging.FileHandler
+    level: INFO
+    formatter: json
+    filename: logs.log
```

Comment:
still need to think about how to set this up; thinking a symlink to a centralized `/var/logs/akatsuki-services/{service}.log`

Reply:
is there not a nice way to use env vars for this?

Reply:
not that i know of

2 hacky ideas:
1. overwrite filename at runtime from env
2. use `envsubst` to turn this into a template where we can inject env vars

Reply:
alternatively, we could commit logging.yaml.example to the repo and have this private and per-install

Reply:
this may actually make the most sense imo as different devs & setups will have different preferences & needs